src/main/resources/application.properties

```diff
@@ -61,6 +61,9 @@ insights.upload.accountnumber=123454
 # Camel configuration
 # tracing enabled?
 camel.springboot.tracing=false
+pagination.flag-assessment.limit.default=100
+pagination.flag-assessment.limit.max=1000
+pagination.report.limit.max=1000
```

Comment:
I think we shouldn't need these variables since we can use https://github.com/project-xavier/xavier-integration/blob/e4293da7eac386a3a82f5ad611a851aebce25daf/src/main/resources/application.properties#L165

Reply:
+1